sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -48,7 +48,7 @@
 - Added async module-level `upload_blob_to_url` and `download_blob_from_url` functions.
 - `ResourceTypes`, and `Services` now have method `from_string` which takes parameters as a string.
 
-## Version 12.0.0b4:
+## 12.0.0b4
```

Comment:
Please add the date, as shown on PyPI:
https://pypi.org/project/azure-storage-blob/#history